site/en/codelabs/telink-openthread-hardware/index.lab.md

```diff
@@ -12,7 +12,7 @@ layout: scrolling
 
 ---
 
-# Build a Thread Network with the TLSR9518 Development Kit and OpenThread
+# Build a Thread Network with the B91 Development Kit and OpenThread
```

Comment:
Sometimes you use "Development Kit" and other times "Development Board".  Suggest it be "Board" throughout the codelab, unless there's a specific reason to use "kit".

Reply:
Your advice is very accurate, so I replaced most of "Development Kit" with "Development Board". However, I retained "kit" in the Telink B91 Development Kit section, as this subsection provides an introduction to the development kit."